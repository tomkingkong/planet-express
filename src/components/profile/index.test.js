import React from 'react';
import { shallow } from 'enzyme';

import { Profile } from '.';

describe('profile component', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Profile />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should send the user back to the previous page', () => {
    const mockFn = jest.fn();
    const history = { goBack: mockFn };
    const wrapper = shallow(<Profile history={history}/>);
    wrapper.find('.profile__back_btn').simulate('click');
    expect(mockFn).toHaveBeenCalled();
  });
});