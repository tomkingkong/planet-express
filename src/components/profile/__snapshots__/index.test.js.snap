// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`profile component should match snapshot 1`] = `
<div
  className="profile"
>
  <button
    className="profile__back_btn"
    onClick={[Function]}
  >
    Back
  </button>
  <h1>
    Under Construction
  </h1>
</div>
`;
